CauldronMods/Controller/Heroes/Gyrosaur/Cards/HiddenDetourCardController.cs

```diff
@@ -338,5 +338,15 @@ private bool IsCardUnder(Card card)
         {
             return Card.UnderLocation.HasCard(card); // AllDetourUnderCards.Contains(card);
         }
+
+        public override CustomDecisionText GetCustomDecisionText(IDecision decision)
+        {
+            return new CustomDecisionText(
+                $"Do you want to put this card beneath Hidden Detour and put {this.Card.UnderLocation.TopCard.Title} into play?",
```

Comment:
I was trying to figure out how to reference the card about to enter play instead of "this card," but couldn't figure it out the with variables available to me. This is likely fine (especially for long Env card names, though inconsistent with the current text: "Do you want to put {card entering play} under Hidden Detour?"

Reply:
Perhaps try
`
var cardName = "this card"
if(decision is YesNoCardDecision yncd)
{
    cardName = yncd.Card.Title;
}`

Reply:
Fantastic. Added.